Travel Scheduler/View Controllers/ScheduleViewController.m

```diff
@@ -118,7 +123,7 @@ - (nonnull __kindof UICollectionViewCell *)collectionView:(nonnull UICollectionV
     NSDate *date = self.allDates[indexPath.item];
     date = removeTime(date);
     NSDate *startDateDefaultTime = removeTime(self.startDate);
-    NSDate *endDateDefaultTime = removeTime(self.endDate);
+    NSDate *endDateDefaultTime = removeTime(self.scheduleEndDate);
```

Comment:
Maybe change the variable name since we do not have an end date default time anymore?